ui.tests/test-module/specs/actions/setFocus/setFocus.runtime.spec.js

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright 2024 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+describe("SetFocus Test", () => {
+    const pagePath = "content/forms/af/core-components-it/samples/setfocustest.html"
+    let formContainer = null
+
+    beforeEach(() => {
+        cy.previewForm(pagePath).then(p => {
+            formContainer = p;
+        });
+    });
+
+    const nextBtn = '#button-c974c7515a-widget';
+    const focusTnC = '#button-61a837277a-widget';
```

Comment:
Can you also add setFocus test case for file attachment ?

Reply:
The test collateral contains file attachment as well and its setFocus is tested.